models/intel/text-spotting-0001-detector/description/text-spotting-0001-detector.md

```diff
@@ -19,37 +19,36 @@ This model is Mask-RCNN-based text detector with ResNet50 backbone and additiona
 
 | Metric                                        | Value     |
 |-----------------------------------------------|-----------|
-| Word spotting hmean ICDAR2015, w/o dictionary | 59.04%    |
+| Word spotting hmean ICDAR2015, without a dictionary | 59.04%    |
 | Detection hmean ICDAR2015                     | 87.09%    |
 | GFlops                                        | 185.169   |
 | MParams                                       | 26.497    |
 | Source framework                              | PyTorch\* |
 
-*Hmean Word spotting* is defined and measured according to
+*Hmean Word spotting* is defined and measured according to the
 [Incidental Scene Text (ICDAR2015) challenge](https://rrc.cvc.uab.es/?ch=4&com=introduction).
 
 ## Performance
 
 ## Inputs
 
-1.	name: `im_data` , shape: [1x3x768x1280] - An input image in the format
-    [1xCxHxW]. The expected channel order is BGR.
-1.	name: `im_info`, shape: [1x3] - Image information: processed image height,
-    processed image width and processed image scale
+1.	Name: `im_data` , shape: [1x3x768x1280]. An input image in the [1xCxHxW] format. 
+    The expected channel order is BGR.
+2.	Name: `im_info`, shape: [1x3]. An input image with a processed image height,
+    processed image width and processed image scale.
     w.r.t. the original image resolution.
 
 ## Outputs
 
-1.	name: `classes`, shape: [100] - Contiguous integer class ID for every
-    detected object, '0' for background, i.e. no object.
-1.	name: `scores`, shape: [100] - Detection confidence scores in range [0, 1]
+1.	Name: `classes`, shape: [100]. Contiguous integer class ID for every
+    detected object, `0` for background (no object detected).
+1.	Name: `scores`, shape: [100]. Detection confidence scores in the [0, 1] range 
     for every object.
-1.	name: `boxes`, shape: [100x4] - Bounding boxes around every detected objects
-    in (top_left_x, top_left_y, bottom_right_x, bottom_right_y) format.
-1.	name: `raw_masks`, shape: [100x2x28x28] - Segmentation heatmaps for all
+1.	Name: `boxes`, shape: [100x4]. Bounding boxes around every detected object
+    in the (top_left_x, top_left_y, bottom_right_x, bottom_right_y) format.
+1.	Name: `raw_masks`, shape: [100x2x28x28]. Segmentation heatmaps for all
     classes for every output bounding box.
-1.  name: `text_features`, shape [100x64x28x28] - Text features that are fed to
-text recognition head.
+1.  Name: `text_features`, shape [100x64x28x28]. Text features fed to a text recognition head.
```

Comment:
Original text is logically correct, output features are input for another type model